datahub-classify/tests/test_similarity_predictor.py

```diff
@@ -222,26 +220,26 @@ def populate_tableinfo_object(dataset_name):
     df = load_df(dataset_name)
     np.random.seed(SEED)
     table_meta_info = {
-        "Name": dataset_name,
-        "Description": f"This table contains description of {dataset_name}",
-        "Platform": PLATFORMS[np.random.randint(0, 5)],
-        "Table_Id": dataset_name,
+        "name": dataset_name,
+        "description": f"This table contains description of {dataset_name}",
+        "platform": PLATFORMS[np.random.randint(0, 5)],
+        "table_id": dataset_name,
     }
     col_infos = []
     for col in df.columns:
         fields = {
-            "Name": col,
-            "Description": f" {col}",
-            "Datatype": str(df[col].dropna().dtype),
-            "Dataset_Name": dataset_name,
-            "Column_Id": dataset_name + "_SPLITTER_" + col,
+            "name": col,
+            "description": f" {col}",
+            "datatype": str(df[col].dropna().dtype),
+            "dataset_name": dataset_name,
+            "column_id": dataset_name + "_SPLITTER_" + col,
         }
-        metadata_col = ColumnMetadata(fields)
+        metadata_col = ColumnMetadata(**fields)
```

Comment:
just construct the ColumnMetadata object directly instead of going through a dict first

same with table_meta_info above

Reply:
I will modify the code accordingly